build.sbt

```diff
@@ -27,6 +27,7 @@ val commonSettings = Seq(
     "Guardian Github Releases" at "https://guardian.github.io/maven/repo-releases",
     "Guardian Github Snapshots" at "https://guardian.github.io/maven/repo-snapshots",
     Resolver.sonatypeRepo("releases"),
+    Resolver.sonatypeRepo("snapshots") //TODO remove before merging!
```

Comment:
like the comment says we should remove this when we are ready to merge. This was just added in order to be able to use a snapshot version of the identity-play-auth library but we don't want to risk leaving snapshot dependencies in prod